examples/show_points_based_on_feature.py

```diff
@@ -4,7 +4,6 @@
 
 .. tags:: visualization-advanced
 """
-#!/usr/bin/env python3
```

Comment:
Why remove this line? I think it should be moved to the beginning of file.

Reply:
`EXE` complains that shebang should only be there if the files is marked as executable. It's the only file that has this in the examples, so this was the minimal change. Alternatively, we could add it to all the files, and mark them all as executable.